components/director/internal/domain/certsubjectmapping/service.go

```diff
@@ -117,6 +107,18 @@ func (s *service) List(ctx context.Context, pageSize int, cursor string) (*model
 	return csmPage, nil
 }
 
+// ListAll retrieves certificate subject mappings
+func (s *service) ListAll(ctx context.Context) ([]*model.CertSubjectMapping, error) {
+	log.C(ctx).Info("Listing certificate subject mappings")
+
+	certSubjectMappings, err := s.repo.ListAll(ctx)
```

Comment:
Currently, we don't have many CSMs, however, if their number increases in the future, getting them all without paging could be nonoptimal. Just a heads-up.